README.md

```diff
@@ -32,6 +34,10 @@ Reference smart contracts to call these functions as well as examples can be fou
 
 For further details on methods, hashes and availability please refer to [PRNG Precompile methods](contracts/util-precompile/README.md)
 
+## Development guidelines
```

Comment:
```suggestion
## Test setup
```

Reply:
you're right! Will push another commit right away!